<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prescription Generator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
  background-color: #1F2544;
}

QLineEdit{
  background-color: #EEEDEB; 
  color: #000000;
  padding: 2px; 
  border:none; 
  border-radius: 5px;
  width: 200px
}

QPushButton{
   background-color: #474F7A;
   border: none; 
   border-radius: 10px;
   padding: 5px; 
   color: #FFD0EC;
}

QPushButton:hover{
  background-color: #343a59
}

QComboBox::editable:!on { border: 4px black; border-style: dotted;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="70,30">
    <item>
     <widget class="QPdfView" name="pdf_viewer" native="true"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="CustomLabel" name="logo">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="doctors_name_label">
            <property name="font">
             <font>
              <family>Academy Engraved LET</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Doctor's name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="doctor_name_line"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="specialist_label">
            <property name="font">
             <font>
              <family>Academy Engraved LET</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Specialist In</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="specialist_line"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="address_label">
            <property name="font">
             <font>
              <family>Academy Engraved LET</family>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="address_line"/>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="patients_name_label">
          <property name="font">
           <font>
            <family>Academy Engraved LET</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Patient's name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="patients_name_line"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="age_label">
          <property name="font">
           <font>
            <family>Academy Engraved LET</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Age</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="age_line"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QComboBox" name="gender_combo">
          <property name="font">
           <font>
            <family>Academy Engraved LET</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="currentText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="date_label">
          <property name="font">
           <font>
            <family>Academy Engraved LET</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="generate_button">
        <property name="font">
         <font>
          <family>Academy Engraved LET</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>customlabel.h</header>
  </customwidget>
  <customwidget>
   <class>QPdfView</class>
   <extends>QWidget</extends>
   <header>qpdfview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
